dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -786,21 +786,21 @@ BEGIN
     IDS_ABOUT_VERSION_STRING "Versiune %S (%S)"
 
     /* context menus */
-    IDS_VIEW_LARGE "D&ale"
+    IDS_VIEW_LARGE "&Dale"
     IDS_VIEW_SMALL "&Pictograme"
     IDS_VIEW_LIST "&Listă"
     IDS_VIEW_DETAILS "&Detalii"
     IDS_SELECT "Selectea&ză"
     IDS_OPEN "&Deschide"
     IDS_CREATELINK "Creează sc&urtătură"
```

Comment:
In the top of the file you changed from
        MENUITEM "Lipește ca sc&urtătură", FCIDM_SHVIEW_INSERTLINK
to
        MENUITEM "Lipire c&omandă rapidă", FCIDM_SHVIEW_INSERTLINK
backed up by your MS pic.

so you must use the same string and accelerator also here for IDS_CREATELINK.

Reply:
I didn't have time to get there. I will fix all the issues these days. This is why I reconvert the PR as a draft. Please wait for a few days until it will be ready for review. I didn't know in the beginning there are so many issues.

Reply:
I will send tomorrow all the screenshots you asked me for.

Reply:
Resolved both to &o.